### PR TITLE
Add recipe for STIM

### PR DESCRIPTION
**Pull request by minnerbe**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by StephanPreibisch** [comment]
Hi, I cannot merge this either

**Comment by StephanPreibisch** [comment]
Hi, I agree to be co-maintainer ...

**Comment by minnerbe** [comment]
@conda-forge-admin, please ping team

**Comment by minnerbe** [comment]
@conda-forge/help-java

**Comment by minnerbe** [comment]
@conda-forge-admin, please lint

File: recipes/stim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: stim
+  version: {{ version }}
+
+source:
+  url: https://github.com/PreibischLab/STIM/archive/refs/tags/v0.3.0-alpha.zip
```
Comment (ocefpaf) [inline recipes/stim/meta.yaml:8]: The versions do not match. You should use a jinja template for the version in the URL. Last but not least, dev release should follow CFEP-05: https://github.com/conda-forge/cfep/blob/main/cfep-05.md 

```suggestion
{% set version = "0.2.0" %}

package:
  name: stim
  version: {{ version }}

source:
  url: https://github.com/PreibischLab/STIM/archive/refs/tags/v0.3.0-alpha.zip
```

**Comment by minnerbe** [comment]
Thanks for reviewing this, @ocefpaf !
After reading the link you provided, I'm realizing that I'm not trying to package a dev-release. The `-alpha` in the original version was referring to possible changes that are necessary for this conda package only. Therefore, I created another release that this package refers to and made the version numbers consistent as you suggested.

I'm happy to include any other suggestions and comments you might have.

**Pull request merged**